origin-dapp/src/css/app.css

```diff
@@ -5519,6 +5580,76 @@ a:not([href]):not([tabindex]).span-link {
   .warning.alert .caret {
     margin-top: -0.15rem;
   }
+
+  .profile-wrapper .container .name h1 {
+    font-size: 1.4rem;
+    line-height: 1.2;
+    font-weight: 300;
+  }
+
+  .profile-wrapper .edit-profile {
+    width: 21px;
+    height: 21px;
+  }
+
+  .profile-wrapper .container .description {
+    font-size: 0.93rem;
+    height: 2.7rem;
+    overflow: hidden;
+  }
+
+  .profile-wrapper .container .description.expanded {
+    height: auto;
+    overflow: auto
+  }
+
+  .profile-wrapper .container .description-options a {
+    font-size: 0.875rem;
+    font-weight: normal;
+  }
+
+  .profile-wrapper #description-caret {
+    padding-left: 0.2rem;
+    padding-bottom: 0.1rem;
+  }
+
+  .profile-wrapper #verify-yourself {
+    font-size: 1rem;
+    margin-bottom: 0.25rem;
+  }
+
+  .services-container {
+    padding: 10px 5px 5px 5px;
+  }
+
+  .services-container .attestation-button {
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+
+  .phone.modal .dropdown .dropdown-item > :last-child {
+    padding-left: 5px;
+  }
+
+  .profile-wrapper .title h2 {
+    margin-bottom: 13px;
+    margin-top: 13px;
+    text-align: center;
+    color: var(--white);
```

Comment:
@sparrowDom can we just use `white` instead of adding a variable?

Reply:
👍 good point